README.md

```diff
@@ -4,254 +4,218 @@
-* Generate new genesis key, and build patched binaries
-(using `build-patched-binaries.sh` script).
-* Run a bunch of nodes and bakers, at least two, but the more the better
-(using `start-baker.sh` script).
-* Customize chain parameters to your taste and activate the procotol
-(using `activate-protocol.sh` script).
-* Play with your chain.
-
-More detailed instructions are presented further.
-
-## Prerequisites
+* Generate a new genesis key, and build patched binaries
+(using the `build-patched-binaries.sh` script).
+* Run a number of nodes and bakers - at least two, but the more the better
+(using the `start-baker.sh` script).
+* Customize the chain parameters and activate the procotol
+(using the `activate-protocol.sh` script).
+* Experiment with the chain.
 
 There are two ways to use these scripts:
-1) Run them as is on your machine.
-2) Run them inside docker container.
+1) Run them inside a docker container.
+2) Run them directly on your local machine.
 
-### Running scripts outside docker prerequisites
+More detailed instructions are presented in the following sections.
 
-Since running a private blockchain requires compiling patched Tezos binaries from
-scratch, you will have to install the following dependencies:
-```
-rsync git m4 build-essential patch unzip bubblewrap wget pkg-config
-libgmp-dev libev-dev libhidapi-dev which opam
-```
-These dependencies are also described [here](https://tezos.gitlab.io/introduction/howtoget.html#build-from-sources).
+## Running the scripts inside Docker
 
-In order to install them using a Debian or RedHat-based distro, run one of the following
-commands:
-```sh
-# Debian and apt
-sudo apt update && sudo apt install rsync git m4 build-essential patch unzip \
-     bubblewrap wget pkg-config libgmp-dev libev-dev libhidapi-dev which
-# RedHat and yum
-sudo yum update && sudo yum install rsync git m4 patch unzip make \
-     bubblewrap wget gmp-devel libev-devel perl-Pod-Html pkgconfig hidapi-devel which
-```
-In order to install `opam`, run the following commands:
-```sh
-wget https://github.com/ocaml/opam/releases/download/2.0.3/opam-2.0.3-x86_64-linux
-sudo cp opam-2.0.3-x86_64-linux /usr/local/bin/opam
-sudo chmod a+x /usr/local/bin/opam
-```
-
-### Running scripts inside docker prerequisites
+### Prerequisites
 
-For this, you obviously will need installed docker.
+We assume you have docker installed and running.
 
-There is [`Dockerfile`](./Dockerfile) that should be used in order to build required docker image
-with all tezos library dependencies.
-Run the following command to do this:
+The file [`Dockerfile`](./Dockerfile) will be used to build a docker image
+with all the required tezos dependencies.
+Run the following command:
 ```sh
 docker build -t ubuntu-tezos .
 ```
 
-In order to use this image you will also need docker volume, to create one run the following command:
+In order to use this image you will also need two docker volumes. Run the following commands:
```

Comment:
Well, that's not actually true for the case when you run only one node

Reply:
Maybe it's better to mention that you need two volumes somewhere at the beginning of "running bakers" section